src/go/cmd/update-collection-v3/migrations/metrics/metrics_test.go

```diff
@@ -0,0 +1,56 @@
+package metrics
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type TestCase struct {
+	inputYaml   string
+	outputYaml  string
+	err         error
+	description string
+}
+
+func runYamlTest(t *testing.T, testCase TestCase) {
+	actualOutput, err := Migrate(testCase.inputYaml)
+	if testCase.err == nil {
+		require.NoError(t, err, testCase.description)
+	} else {
+		require.Equal(t, err, testCase.err, testCase.description)
+	}
+	require.Equal(t, strings.Trim(testCase.outputYaml, "\n "), strings.Trim(actualOutput, "\n "), testCase.description)
+}
+
+func Test_EventsEnabledProvider(t *testing.T) {
```

Comment:
This test does not follow the convention described in point 4. of https://github.com/SumoLogic/sumologic-kubernetes-tools/tree/v2.14.0/src/go/cmd/update-collection-v3#how-to-add-a-new-migration. Wouldn't it be nice if it did? :wink: